### PR TITLE
[HttpKernel] add @var tag to container

### PR DESCRIPTION
**Pull request by BASAKSemih**
| Q             | A
| ------------- | ---
| Branch?       | 6.2 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       |  
| License       | MIT
| Doc PR        | n/a 

Hello just a chore commit to add a @var tag.

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php [hunk 1]
```diff
@@ -23,6 +23,9 @@
  */
 class ContainerControllerResolver extends ControllerResolver
 {
+    /**
+     * @var ContainerInterface
+     */
```
Comment (OskarStark) [inline src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php:28]: The shouldn't be needed, as it's type hinted in the constructor

Comment (BASAKSemih) [reply src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php:28]: Hi, i can suggest `protected ContainerInterface $container;`

Comment (stof) [reply src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php:28]: Changing a protected property to be typed is a BC break (for child classes the redeclare it)

**Comment by derrabus** [comment]
We don't add these annotations for the reason @OskarStark explained. Closing therefore, but thank you for your proposal.

**Pull request closed without merging**